packages/react-scripts/template/src/App.test.js

```diff
@@ -1,5 +1,7 @@
+import 'jest';
```

Comment:
An alternative to adding this import is to add `types: [ 'jest' ]` to tsconfig.json

Reply:
I don't believe either should be needed. Typescript should pick up the `node_modules/@types` definitions automatically without any extra effort.